package.json

```diff
@@ -76,5 +77,8 @@
     "lint": "standard",
     "test": "echo 'not yet implemented, wanna help?'"
   },
+  "standard": {
+    "parser": "babel-eslint"
```

Comment:
Btw, you can remove that depency by chaning your `lint` script to be `standard --parser babel-eslint` but you had to rely on having `babel-eslint` on the global scope, just as I assume you have standard as well :)